app/assets/stylesheets/src/shared/_components.scss

```diff
@@ -175,8 +175,8 @@
   &.tiny {
     --sc_font_size: 12px;
     --sc_min_height: 28px;
-    --sc_chevron_size: 0.4em;
-    --sc_check_size: 0.7em;
+    --sc_chevron_size: 5.3px;
+    --sc_check_size: 8.84px;
```

Comment:
for icon calculate this value [here](https://www.calculator.net/triangle-calculator.html?vc=45&vx=&vy=6.25&va=45&vz=&vb=90&angleunits=d&x=Calculate)

Reply:
![image](https://github.com/user-attachments/assets/12161b6f-d496-4c00-b886-6bf07d9e895a)